stratum/lib/utils.h

```diff
@@ -247,6 +249,28 @@ inline U ByteStreamToUint(const std::string& bytes) {
   return val;
 }
 
+// Create secure TLS gRPC channel credentials for use in clients.
+inline ::util::StatusOr<std::shared_ptr<::grpc::ChannelCredentials>>
+CreateSecureClientGrpcChannelCredentials(
```

Comment:
```suggestion
CreateGrpcClientChannelCredentials(
```

We have the same code block in all usages:
```c++
  std::shared_ptr<::grpc::ChannelCredentials> channel_credentials;
  if (!FLAGS_ca_cert.empty()) {
    ASSIGN_OR_RETURN(channel_credentials,
                     CreateSecureClientGrpcChannelCredentials(
                         FLAGS_client_key, FLAGS_client_cert, FLAGS_ca_cert));
  } else {
    channel_credentials = ::grpc::InsecureChannelCredentials();
  }
```
I would suggest moving the if logic into the helper and simplifying the tools.

Reply:
Done, now part the cred manager